datajunction-server/datajunction_server/api/sql.py

```diff
@@ -123,11 +118,9 @@ async def get_sql(  # pylint: disable=too-many-locals
     session: AsyncSession = Depends(get_session),
     engine_name: Optional[str] = None,
     engine_version: Optional[str] = None,
-    current_user: Optional[User] = Depends(get_current_user),
-    validate_access: access.ValidateAccessFn = Depends(  # pylint: disable=W0621
-        validate_access,
-    ),
-) -> TranslatedSQL:
+    current_user: Optional[User],
```

Comment:
It's because the `Depends(...)` thing is a special FastAPI functionality, and it's only available for API endpoints. Moving it out to a helper function isn't an API endpoint anymore and so `Depends` won't work the same way